qiskit_experiments/curve_analysis/composite_curve_analysis.py

```diff
@@ -251,22 +247,22 @@ def _run_analysis(
             if self.options.plot and analysis.options.plot_raw_data:
                 for model in analysis.models:
                     sub_data = processed_data.get_subset_of(model._name)
-                    self.drawer.draw_raw_data(
-                        x_data=sub_data.x,
-                        y_data=sub_data.y,
-                        name=model._name + f"_{analysis.name}",
+                    self.plotter.set_series_data(
```

Comment:
Should this be called `add_series_data`?

Reply:
I did think about using `add_series_data` and `add_figure_data` but chose `set_*` as repeated calls to `set_*_data` with the same data-keys will overwrite the associated data in the plotter. `add_*` would imply that there is a collection of entries per series-name and data-key, and that the data is not overwritten; which didn't match up with prior usage of `BaseCurveDrawer`. The original drawer class and the analysis classes that use it identify data as single NumPy array variables (`x`, `y_err`, `y_mean`), per series-/model-name, and not lists of NumPy arrays. If we want `BasePlotter` to accept multiple NumPy arrays per data-key without overwriting previous values, then I do think the functions should be renamed to `add_*_data`. Though in this case we would need to define how a subclass should treat multiple data entries for the same data-key.

Reply:
Ok in this case `set` makes sense.